From: Ramez Medhat <iramezdev@gmail.com>
Date: Wed, 21 Jan 2026 17:28:22 +0200
Subject: quickSettingsMenu: Add actor group to focus manager

This allows focus to menu item using the tab key to match the behaviour in other menus.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/4043>
(cherry picked from commit 92eebde81cebfc25bdd4ef5ec1815f59598ba6bd)
Bug: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/8904
Origin: upstream, 49.4, commit:cf368daa85028bac2e3b636aef7b9445973d3615
---
 js/ui/quickSettings.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/js/ui/quickSettings.js b/js/ui/quickSettings.js
index 0512948..ed6c471 100644
--- a/js/ui/quickSettings.js
+++ b/js/ui/quickSettings.js
@@ -734,6 +734,8 @@ export const QuickSettingsMenu = class extends PopupMenu.PopupMenu {
         this.actor.add_child(this._boxPointer);
         this.actor._delegate = this;
 
+        global.focus_manager.add_group(this.actor);
+
         this.connect('menu-closed', () => this.actor.hide());
 
         Main.layoutManager.connectObject('system-modal-opened',
